src/tiledimage.js

```diff
@@ -1780,21 +1816,18 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
             levelVisibility
         );
 
-        if (!tile.loaded) {
-            if (tile.context2D) {
-                this._setTileLoaded(tile);
-            } else {
-                var imageRecord = this._tileCache.getImageRecord(tile.cacheKey);
-                if (imageRecord) {
-                    this._setTileLoaded(tile, imageRecord.getData());
-                }
-            }
+        if (!tile.loaded && !tile.loading) {
+            // Tile was created or its data removed: check whether cache has the data.
+            // this method sets tile.loading=true if data available, which prevents
+            // job creation later on
+            this._tryFindTileCacheRecord(tile);
         }
 
         if ( tile.loading ) {
             // the tile is already in the download queue
             this._tilesLoading++;
-        } else if (!loadingCoverage) {
+        } else if (!tile.loaded && !loadingCoverage) {
```

Comment:
I was having a zombie cache test to check no covered (coverage) tile is being downloaded again, which was passing before the merge and now did not and I had to add this check. I am bit perplexed why this was the case since `loadingCoverage` includes the tile loaded check as well, but this really fixed the test. Weird. I should remember this and try to fixure this out..